packages/development-proxy/preset.js

```diff
@@ -1,3 +1,28 @@
 module.exports = {
   mixins: [__dirname],
+  configSchema: {
+    proxy: {
+      oneOf: [
+        {
+          type: 'string',
+          format: 'uri',
+        },
+        {
+          type: 'object',
+          patternProperties: {
+            '^/': {
+              oneOf: [
+                { type: 'string', format: 'uri' },
+                {
+                  type: 'object',
+                  properties: { target: { type: 'string', format: 'uri' } },
+                },
+              ],
+            },
+          },
+          additionalProperties: false,
+        },
+      ],
```

Comment:
Does this make the proxy configuration required? It is optional at the moment.

Reply:
No, all properties are optional unless specified otherwise (see [required properties](https://json-schema.org/understanding-json-schema/reference/object.html#required))